lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -381,13 +381,15 @@ def point_of_contact
       def translate_treatments
         return {} if input_form['vaTreatmentFacilities'].blank?
 
+        # treatmentCenterName clean up is an approximation of evss regex
+        # validation ([a-zA-Z0-9"\/&\(\)\-'.#]([a-zA-Z0-9(\)\-'.# ])?)+$
```

Comment:
```suggestion
    # EVSS validates `treatmentCenterName` using the regex: `([a-zA-Z0-9"\/&\(\)\-'.#]([a-zA-Z0-9(\)\-'.# ])?)+$`, so
    # we make a best effort to clean up `treatmentCenterName` so that it passes this regex
```

Am I understanding your comment correctly? I made it a bit more verbose to make it clearer.